src/textual/css/stylesheet.py

```diff
@@ -8,17 +8,13 @@
 
 import rich.repr
 from rich.console import RenderableType, RenderResult, Console, ConsoleOptions
-from rich.highlighter import ReprHighlighter
```

Comment:
Removing some unused imports while I'm there

(I think it could definitely be nice to add Pylint to the project, so it could spot those across the whole codebase automatically - even if it's to disable all its default rules and enable only this one?)